samples/event-logger/src/index.js

```diff
@@ -1,19 +1,18 @@
-import {render} from 'react-dom';
+import {createRoot} from 'react-dom';
 import {Provider} from 'react-redux';
 
 import App from './App';
 import storeFactory from './store';
 
-const store = storeFactory();
-const appElement = (
```

Comment:
was the removal of this variable mandatory? Same question for all the other sample apps